installer/installer.go

```diff
@@ -135,6 +135,13 @@ func (i *Installer) getLibDir() string {
 		return env
 	}
 
+	if runtime.GOOS == "windows" {
+		if out, err := exec.Command("powershell", "-Command", "(Get-Command gcc).Source").Output(); err == nil && len(out) > 0 {
```

Comment:
Does this error need handling?

Reply:
Not sure. I guess the only sane thing I could do in this case is log the error. What do you think? Is it worth logging? The reason might simpoly be that `gcc` is not found on the system.